config/database.yml

```diff
@@ -81,5 +81,5 @@ test:
 production:
   <<: *default
   database: personal-finance_production
-  username: personal-finance
-  password: <%= ENV['PERSONAL-FINANCE_DATABASE_PASSWORD'] %>
+  username: <%= ENV['FINANCE_DATABASE_USERNAME'] %>
+  password: <%= ENV['FINANCE_DATABASE_PASSWORD'] %>
```

Comment:
@marderer how can other developers understand which env variables are used in the app? 
`.env`-variables pattern goes with sample file usually

```
/.env_sample

FINANCE_DATABASE_USERNAME=''
FINANCE_DATABASE_PASSWORD=''
```

When new developer will start working on this project he can open `.env_sample` and see which env variables he/she has to setup.

P.S. will be cool to specify usage of env variables with a link to `env_sample` in README.